tests/quickchecks.rs

```diff
@@ -227,4 +234,32 @@ quickcheck! {
             assert_eq!(val, alloc);
         }
     }
+
+    fn all_allocations_in_a_chunk(values: Vec<BigValue>) -> () {
+        let b = Bump::new();
+        let allocated: Vec<&BigValue> = values.into_iter().map(|val| b.alloc(val) as &_).collect();
+        let chunks: Vec<(*mut u8, usize)> = unsafe { b.iter_allocated_chunks_raw() }.collect();
+        for alloc in allocated.into_iter() {
+            assert!(chunks.iter().any(|&(ptr, size)| {
+                let ptr = ptr as usize;
+                let chunk = (ptr, ptr + size);
+                contains(chunk, range(alloc))
+            }));
+        }
+    }
+
+    fn chunks_and_raw_chunks_are_same(values: Vec<BigValue>) -> () {
+        let mut b = Bump::new();
+        for val in values {
+            b.alloc(val);
+        }
+        let raw_chunks: Vec<(_, _)> = unsafe { b.iter_allocated_chunks_raw() }.collect();
+        let chunks: Vec<&[_]> = b.iter_allocated_chunks().collect();
```

Comment:
Can you also add `assert_eq!(chunks.len(), raw_chunks.len())` here? `zip` will silently ignore elements from one of the iterators if they aren't both the same size.